documentation/docs/guide/content/using-custom-post-types.md

```diff
@@ -0,0 +1,206 @@
+# Using Custom Post Types
+
+Welcome to this tutorial on using custom post types with the Block Theme. The goal of this tutorial is to show you how to setup and use a custom post type from your WordPress dashboard. But before we get into that, let's talk about custom post types and why they're useful.
+
+## What are custom post types?
+
+### Introduction
+
+A custom post type is used when the regular WordPress posting system is not suitable for a task. The regular WordPress posting system allows you to create news/blog posts and organize/manage them from the WordPress Dashboard. On the public side of your website, these posts can then be displayed to the user as a feed. This functionality is ideal for simple scenarios where you wish to keep users up to date. However, maybe you require an events calendar that sorts by event date, or a product catalogue. Perhaps you would like to use the regular posting system for news/blog updates but need a different post type for a different purpose. This is when custom post types become useful.
```

Comment:
Suggested change from:
.... However, maybe you require an ....
to:
.... However, you may require an ....

Consider changing when to where. From
.... This is when custom... 
to
.... This is where custom